assets/tests/components/notificationModals/chelseaRaisedNotificationModal.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { mount } from "enzyme"
```

Comment:
**issue:** when adding new tests we should use React Testing Library since Enzyme is no longer well-supported (and I'd like to gradually transition our old tests over).

Reply:
fixed in 15a8e5f2